### PR TITLE
Fix link to macros chapter

### PR DESCRIPTION
**Pull request by steveklabnik**
We're gonna link to nightly as this didn't make it into the corresponding stable.

r? @Mark-Simulacrum @Centril

### CODE REVIEW

**Comment by Mark-Simulacrum** [comment]
Hm, the link points to stable?

File: RELEASES.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ Misc
 [cargo/5878]: https://github.com/rust-lang/cargo/pull/5878/
 [cargo/5984]: https://github.com/rust-lang/cargo/pull/5984/
 [cargo/5995]: https://github.com/rust-lang/cargo/pull/5995/
-[proc-macros]: https://doc.rust-lang.org/book/2018-edition/ch19-06-macros.html
+[proc-macros]: https://doc.rust-lang.org/stable/book/2018-edition/ch19-06-macros.html
```
Comment (frewsxcv) [inline RELEASES.md]:
```suggestion
[proc-macros]: https://doc.rust-lang.org/nightly/book/2018-edition/ch19-06-macros.html
```

**Review by frewsxcv — COMMENTED** [review]
r=me with the `stable` → `nightly` adjustment

**Comment by steveklabnik** [comment]
dangit, I must have made an error, sorry about that! anyway applied

**Pull request merged**